ImageCropper/ImageCropper.iOS/ImageCropperImplementation.cs

```diff
@@ -32,6 +32,8 @@ public void ShowFromFile(ImageCropper imageCropper, string imageFile)
             if(imageCropper.AspectRatioX > 0 && imageCropper.AspectRatioY > 0)
             {
                 cropViewController.AspectRatioPreset = TOCropViewControllerAspectRatioPreset.Custom;
+                cropViewController.ResetAspectRatioEnabled = false;
+                cropViewController.AspectRatioLockEnabled = true;
                 cropViewController.CustomAspectRatio = new CGSize(imageCropper.AspectRatioX, imageCropper.AspectRatioY);
```

Comment:
Which issues is there?

Reply:
When set the Custom AspectRatio (for example: X = 1, Y = 1), in Android all is well, the aspectRatio is maintained while the user cropping the picture. But in IOS the user can change the aspectRatio, simply dragging the clipping area or in the configuration bar

Reply:
Thanks for your contribution. :)

Reply:
You are welcome

Reply:
here is another library that provide aspect ratio in iOS. can you not used in this project. Below is link: 
https://github.com/nkopilovskii/ImageCropper